audio/param.rs

```diff
@@ -263,7 +263,11 @@ pub(crate) enum AutomationEvent {
     SetValueAtTime(f32, Tick),
     RampToValueAtTime(RampKind, f32, Tick),
     SetTargetAtTime(f32, Tick, /* time constant, units of Tick */ f64),
-    SetValueCurveAtTime(Vec<f32>, Tick, /* duration */ Tick),
+    SetValueCurveAtTime(
+        Vec<f32>,
+        /* start time, units of Tick */ Tick,
+        /* duration, units of Tick */ Tick,
```

Comment:
(no need to clarify the unit, the type is the unit)

Reply:
Hi Manish,

Thank you for the comments, I don't quite understand how to implement the k_float stuff. Do you want me to get rid of the step variable and just store (current_tick.0 as f32) - (start.0 as f32) in t_k (which will be progress * duration) and something else in t_k_next? If so, I don't understand how I can calculate the t_k_next value without using the step value.

Can you help me out with this?

Thanks.

Reply:
You don't need `time_k_next`

You already have `k_float - k`, which is a number 0 to 1 representing where to interpolate.

So you can do something like

```rust
if k + 1 < n {
   let progress = k_float - k
   *value = v[k] * (1 - progress) + v[k + 1] * progress;
} else {
  *value = v[n - 1];
}
```

Reply:
Hi Manish,

Thank you. I have made the changes and ran rustfmt. Is it fine now?